lib/llvm-backend/src/code.rs

```diff
@@ -1741,7 +1745,12 @@ impl FunctionCodeGenerator<CodegenError> for LLVMFunctionCodeGenerator {
                             _ => value,
                         });
                     }
-                    _ => unimplemented!("multi-value returns"),
```

Comment:
Do you really need this change? AFAICT this is a correct use of unimplemented. The WASM feature "multi-value returns" really is unimplemented in our VM (indeed, not just here, but throughout).

(Applies to the other mentions of multi-value return in this change.)

Reply:
You totally right, it's a legitimate use of `unimplemented!`. 

I decided to replaced them because the function already return `Result` with `CodegenError`.
The cost for us to replace them by a returning error is nothing compared to having wasmer panicking if we use `unimplemented!` or even `unreachable!`.

Reply:
I know these are unreachable, the parser is not supposed to accept multi-value return yet because it is an extension and we have not enabled it. It's a serious bug if this is reached in normal usage.

The benefit to keeping them is that when someone comes along to implement multi-value return (or another unimplemented feature) these are easily marked as such.

Can I ask what the downside is of letting this panic? It's bad for embedders if a library it's using panics, but as I stated that really shouldn't happen (or, if it does, I think this is worth panicking over!). Is it causing problems for the fuzzing effort?

Reply:
Ok make sense, in that case i will convert `unimplemented!` to `unreachable` only for the ones related to multi-value return.

Regarding fuzzing, a panic generate an exit status `101` and an Error will be an exit status `1`.
This difference impact the bug classification of some fuzzers and make bugs triaging more complicated. Also, having too much panic triggered reduce the fuzzing speed since a panic will force the fuzzer to kill the current fuzzing thread and create a new one.